cmd/metricswatcher/README.md

```diff
@@ -0,0 +1,21 @@
+# metricswatcher
+
+`metricswatcher` connects to a Prometheus instance of Mattermost and watches some metrics.
+
+## Configuration
+
+Check the `PrometheusConfiguration` section of your config file:
+
+```json
+    "PrometheusConfiguration": {
+        "PrometheusURL": "http://localhost:9090",
+        "UpdateIntervalInMS": 1000
+    }
+```
+
+## Running
+
+```
+metricswatcher --config config/config.default.json --queries config/prometheusqueries.sample.json
```

Comment:
This also will need to change of course.

Reply:
Updated. I also updated the configuration part, just explaining where to set configuration instead of showing the possible settings (it became too big for a README file).